### ISSUE TITLE
[i] SC 1.3.5 "Identify Input Purpose" - auto-complete is not the primary objective of this requirement

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-wcag-videos#69]
This script is based on a [persona from the published WAI resource "What's New in WCAG 2.1"](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#135-identify-input-purpose-aa). Yet several [comments in the thorough review survey](https://www.w3.org/2002/09/wbs/1/WCAGvideos_batch2thorough/results#xq4) indicate that auto-complete is not the main purpose of this requirement, it is actually a by-product. Is it still good to keep this focus because it is more distinct from the [script for SC 1.3.6](https://wai-wcag-videos.netlify.app/sc-1.3.6/)?

**Comment by detlevhfischer** [comment]
As I said in my survey comment, I think it fudges the issue to focus on autocomplete here because this is orthogonal to the declared intention of 1.3.5. If the chicken-and-egg argument is proven to have failed (the chicken has been put into the world, but it has failed to lay any eggs for several years) I think it is time to retire the chicken.
